package.json

```diff
@@ -21,9 +22,16 @@
     "babel-loader": "^8.2.2",
     "clean-webpack-plugin": "^3.0.0",
     "cross-env": "^7.0.3",
+    "css-loader": "^5.1.0",
+    "file-loader": "^6.2.0",
```

Comment:
I've noticed you didn't use this loader in your config file

Reply:
done